templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,94 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL Content Item.
+   */
+  Parameters:
+    - "content_item" (associative array) (default: predefined structure): format:
+        {
+          description: '', (string) (default: ''): Description of Content Item
+          meta: [], (array) (default: []): Meta's for the Content Item
+          title: {}, (associative array) (default: {}): Predefined structure compatible with Link component. If Content Item type is a 'tile', only label property is required.
+          image: '', (string) (default: ''): Url/path to background image (non required if Content Item type is a 'tile')
+          image_position: 'after' (string) (Can be 'before' or 'after'. Is required only if Icon is specified)
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _content_item = {
+  type: 'default',
+  description: '',
+  meta: [],
+  title: {},
+  image: '',
+  image_position: 'before',
+  links: []
+} %}
+
+{% if content_item is defined %}
+  {% set _content_item = _content_item|merge(content_item) %}
+{% endif %}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Create macro #}
+
+{% macro image(src, alt, class1, class2) %}
+  <div role="img" aria-label="{{ alt }}" class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl {{ class1 }}"
+       style="background-image:url(&quot;{{ src }}&quot;);background-size:contain;height:5rem;width:7.5rem"></div>
+  <div role="img" aria-label="{{ alt }}"
+       class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-none {{ class2 }}"
+       style="background-image:url(&quot;{{ src }}&quot;);background-size:contain;height:8.75rem;width:13.125rem">
```

Comment:
The `&quot;` must be removed, it is probably a rendering bug of the ECL UI thingie. According the [W3 specification](https://www.w3.org/TR/CSS21/syndata.html#uri) quotes can be double `"`, single `'`, or not present at all, and more importantly, the URL inside must be escaped for a few special characters, when not using quotes:

> Some characters appearing in an unquoted URI, such as parentheses, white space characters, single quotes (') and double quotes ("), must be escaped with a backslash so that the resulting URI value is a URI token: '\(', '\)'. 

I suggest to use single quotes and to escape parenthesis and double quotes.

Reply:
:+1: